sdk/core/azure-core/pom.xml

```diff
@@ -84,7 +84,7 @@
     <dependency>
       <groupId>io.projectreactor</groupId>
       <artifactId>reactor-core</artifactId>
-      <version>3.3.5.RELEASE</version> <!-- {x-version-update;io.projectreactor:reactor-core;external_dependency} -->
+      <version>3.3.8.RELEASE</version> <!-- {x-version-update;io.projectreactor:reactor-core;external_dependency} -->
```

Comment:
Could you please update the changelog and call out the version update for `azure-core`, `azure-core-http-netty` and `azure-cosmos` modules?

Reply:
@srnagar - sure will do that once the cosmos live test CI finishes.

Reply:
@srnagar - done, please check.